webapp/public/js/domjudge.js

```diff
@@ -457,31 +457,28 @@ function enableRefresh($url, $after, usingAjax) {
             }).done(function(data, status, jqXHR) {
                 processAjaxResponse(jqXHR, data);
                 $('.loading-indicator').removeClass('ajax-loader');
+                refreshHandler = setTimeout(refresh, $after * 1000);
             });
         } else {
             window.location = $url;
         }
     };
-    if (usingAjax) {
-        refreshHandler = setInterval(refresh, $after * 1000);
-    } else {
-        refreshHandler = setTimeout(refresh, $after * 1000);
-    }
+    refreshHandler = setTimeout(refresh, $after * 1000);
     refreshEnabled = true;
     setCookie('domjudge_refresh', 1);
 
     if(window.location.href == localStorage.getItem('lastUrl')) {
-	window.scrollTo(0, localStorage.getItem('scrollTop'));
+        window.scrollTo(0, localStorage.getItem('scrollTop'));
     } else {
-	localStorage.setItem('lastUrl', window.location.href);
-	localStorage.setItem('scrollTop', 42);
+        localStorage.setItem('lastUrl', window.location.href);
+        localStorage.setItem('scrollTop', 42);
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/mvr320/domjudge/pullRequest?prid=8427924)
```suggestion
	localStorage.setItem("scrollTop", 42);
```